azure_functions_worker/bindings/meta.py

```diff
@@ -70,22 +68,34 @@ def from_incoming_proto(
     else:
         metadata = {}
 
+    pb_type = pb.WhichOneof('rpc_data')
+    if pb_type == 'rpc_shared_memory':
+        # Data was sent over shared memory, attempt to read
+        datum = datumdef.Datum.from_rpc_shared_memory(pb.rpc_shared_memory,
+                                                      shmem_mgr)
+    elif pb_type == 'data':
```

Comment:
Is `pb_type == 'data'` the more common case? If yes, then could you please make that the first in the if condition.